roles/lib_openshift/library/oc_objectvalidator.py

```diff
@@ -1380,7 +1380,14 @@ def _is_invalid_namespace(namespace):
                 all_invalid[invalid_msg] = invalid
 
         if failed:
-            return {'failed': True, 'msg': 'All objects are not valid.', 'state': 'list', 'results': all_invalid}
+            return {
+                'failed': True,
+                'msg': "All objects are not valid.  If you are a supported customer please contact \
+                        Red Hat Support providing the complete output above. If you are not a customer \
+                        please contact users@lists.openshift.redhat.com for assistance.",
```

Comment:
@ashcrow pro-tips from a python masta on how to handle long lines without introducing formatting?